common/constant/key.go

```diff
@@ -416,3 +416,10 @@ const (
 	MetricsMetadata = "dubbo.metrics.metadata"
 	MetricApp       = "dubbo.metrics.app"
 )
+
+// default meta cache config
+const (
+	DefaultMetaCacheName = "dubbo.meta"
+	DefaultMetaFileName  = "dubbo.metadata"
+	DefaultEntrySize     = 100
+)
```

Comment:
These lines of code should not belong to this patch. Please pull from the latest main branch and rebase onto it.

Reply:
done